addons/survey/views/survey_question_views.xml

```diff
@@ -200,8 +200,8 @@
                                     <field name='comment_count_as_answer'
                                         attrs="{'invisible': ['|', ('question_type', 'not in', ['simple_choice','multiple_choice', 'matrix']), ('comments_allowed', '=', False)]}"/>
                                 </group>
-                                <group string="Layout">
-                                    <field name="is_conditional" attrs="{'invisible': [('questions_selection', '=', 'random')]}"/>
+                                <group string="Layout" attrs="{'invisible': [('questions_selection', '=', 'random')]}" >
+                                    <field name="is_conditional"/>
```

Comment:
Looks a bit weird when this is invisible but I guess we can't do much about it unless you have an idea :)

Reply:
Hello @thiryr 😄 

I had thought of one solution for that but didn't apply it as per my reviewer's advice, which is to remove the individual groups of each row and merge both the group in one, so that way if this(2nd) group is invisible the 3rd group will take its place.

see this commit for ref. : ac6348f2

Thanks for the Validation!!

Reply:
Mm, seems fine to me. The current groups seem oddly arbitrary, and this doesn't seem to break anything.

Though maybe your reviewer had a good argument against it :)
Otherwise, I'm fine with keeping everything in the same group

Reply:
Indeed, you are right that it looks a bit weird but let's see wait for LNA's opinion on this while BE Testing. till then Greening it so he can take it in BE Testing. 😃